doc/source/serve/handle-guide.md

```diff
@@ -58,4 +58,11 @@ In both types of ServeHandle, you can call a specific method by using the `.meth
 :start-after: __begin_handle_method__
 :end-before: __end_handle_method__
 :language: python
-```
\ No newline at end of file
+```
+
+## Note about metrics
```

Comment:
Can we reformat this section (and the wording) to:

```markdown
:::{note}
Different metrics are collected when Deployments are called
via Python `ServeHandles` and when they are called via HTTP.

See the [Ray Serve monitoring documentation](serve-monitoring) for more details.
:::
```

That syntax renders notes that look like this:

<img width="767" alt="Screen Shot 2022-08-15 at 10 09 04 AM" src="https://user-images.githubusercontent.com/92341594/184682047-e6cbc449-b836-4463-a51b-760c68644268.png">

Reply:
Ok, updated.